selene-lib/tests/lints/mismatched_arg_count/reassigned_variables_2.stderr

```diff
@@ -11,7 +11,7 @@ error[mismatched_arg_count]: this function takes 3 arguments but 4 arguments wer
    │ ^^^^^^^^^^^^^^^ expected 3 arguments
 
 error[mismatched_arg_count]: this function takes 3 arguments but 4 arguments were supplied
-   ┌─ reassigned_variables_2.lua:1:16
+   ┌─ reassigned_variables_2.lua:13:1
```

Comment:
I don't understand, these are changing to `13:1`, but the line number in the stderr still says `1`.

Reply:
![image](https://user-images.githubusercontent.com/3176481/214471394-a1b0253a-db30-4e0d-b3c0-dad27854ffad.png)


Can't reproduce.
Working my env right. Maybe confuse with the `warning` part?

Reply:
Maybe we're just passing the wrong things into codespan then :-(